voice/bxml/verbs/conference.md

```diff
@@ -23,7 +23,15 @@ A conference may last for at most 24 hours.
 
 BXML returned by callbacks will be queued and executed in the conference in the order they are received, never overlapping. If an error happens when executing a verb, it will continue to the next verb normally.
 
-Only `<PlayAudio>` and `<SpeakSentence>` are supported for conferences. BXML containing unsupported verbs will be rejected completely.
+Only the following verbs are valid for conferences:
+* [`PlayAudio`](playAudio.md)
+* [`SpeakSentence`](speakSentence.md)
+* [`StartRecording`](startRecording.md)
+* [`StopRecording`](stopRecording.md)
+* [`PauseRecording`](pauseRecording.md)
+* [`ResumeRecording`](resumeRecording.md)
+
+BXML containing unsupported verbs will be rejected completely.
```

Comment:
There's a list of callbacks at the end of this file, might want to include the new callback there too.

Reply:
I reasoned against it on the grounds that it isn't this that produces it, but the `StartRecording` verb

Reply:
Oh, makes sense.